site/docs/v1/tech/apis/_webauthn-response-body-base.adoc

```diff
@@ -0,0 +1,77 @@
+[api]
+ifeval::["{base_field_name}" != "credential"]
+[field]#{base_field_name}# [type]#[Array]#::
+The list of WebAuthn Passkey objects.
+endif::[]
+
+[field]#{base_field_name}.algorithm# [type]#[Long]#::
+The identifier for the signing algorithm used with the passkey. These values are defined by https://www.rfc-editor.org/rfc/rfc9053[RFC 9053]. FusionAuth supports a subset of these algorithms.
+
+[field]#{base_field_name}.attestationType# [type]#[String]#::
+The type of attestation provided when the passkey was registered. Supported values include:
++
+ * `none` - no attestation was provided
```

Comment:
This conflicts with the import API above, does it not?

That says that any value will be stored, where this says that `none` is the only supported value.

What am I missing?

Reply:
FusionAuth does not currently support attestation during passkey registration, but if the passkey is imported from another system that _did_ support attestation during registration, it may be useful for customers to track that attestation was provided.

Reply:
So are you saying that if someone creates a credential in FusionAuth, the value will be `none`, but if it is imported from elsewhere, the value may be something else? That makes sense, but I don't get this from the desc of this field. I'd probably just drop the supported values sentence/list. If you don't want to do this, I'd explain what you just said: if the passkey was created in FusionAuth, the value of this will be `none`, but if it was imported, it can be whatever the importer provided.